observation_portal/requestgroups/serializers.py

```diff
@@ -302,22 +304,16 @@ def validate(self, data):
             raise serializers.ValidationError(_(f'Multiple instrument configs are not allowed for type {data["type"]}'))
 
         # Validate the guide mode
-        guide_validation_helper = ModeValidationHelper('guiding', instrument_type, default_modes, modes)
-        if guide_validation_helper.mode_is_not_set(guiding_config):
-            guide_mode_to_set = guide_validation_helper.get_mode_to_set()
-            if guide_mode_to_set['error']:
-                raise serializers.ValidationError(_(guide_mode_to_set['error']))
-            if guide_mode_to_set['mode']:
-                guiding_config['mode'] = guide_mode_to_set['mode']['code']
-            else:
-                guiding_config['mode'] = GuidingConfig.OFF
-
-        guiding_config, guide_mode_error_msg = guide_validation_helper.get_validated_config_and_error(
-            guiding_config['mode'], guiding_config
-        )
-        data['guiding_config'] = guiding_config
-        if guide_mode_error_msg:
-            raise serializers.ValidationError(_(guide_mode_error_msg))
+        guide_validation_helper = ModeValidationHelper('guiding', instrument_type, modes['guiding'])
+
+        guiding_config = guide_validation_helper.validate(guiding_config)
+        if not guiding_config.get('mode'):
+            # Guiding modes have an implicit default of OFF (we could just put this in all relevent validation_schema)
```

Comment:
Already as it is, I think the mode will be set at this point already because the modes in configdb generally will all have defaults set. Is this what you are referring to? I do agree that getting the default from configdb sounds better than setting this here in the code.

Reply:
At least for our unit tests configdb data that wasn't the case, so I left this in since this is what the code used to do. I'm not sure if we should remove it, because on the one hand it is useful to have a sane default (guiding mode now defaults to empty string), but on the other hand it would be nice to remove this and rely entirely on configdb being setup 'correctly'

Reply:
Do you remember why we set the default guide mode to an empty string? I'm wondering if we can set it to `GuidingConfig.OFF` instead.